### PR TITLE
Ch2 figures

### PR DESCRIPTION
**Pull request by ardalis**
## Summary

Fixing 2-x figure numbers with actual numbers.

### CODE REVIEW

File: docs/architecture/cloud-native/deploy-containers-azure.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ Containers provide many benefits, one of which is portability. You can easily ta
...
+Azure Container Registry lets you build, store, and manage images for all of your container deployments. You can [create a container registry using the Azure Portal](https://docs.microsoft.com/azure/container-registry/container-registry-get-started-portal) or [using the Azure CLI](https://docs.microsoft.com/azure/container-registry/container-registry-get-started-azure-cli) or [PowerShell tools](https://docs.microsoft.com/azure/container-registry/container-registry-get-started-powershell). Creating a new container registry just requires an Azure subscription, a resource group, and a unique name. Figure 2-13 shows the basic options for creating a registry, which will be hosted at *registryname*.azurecr.io.
 
 ![Create container registry](media/create-container-registry.png)
```
Comment (rpetrusha) [inline docs/architecture/cloud-native/deploy-containers-azure.md]:
```suggestion
![Create container registry](media/create-container-registry.png) \
```

File: docs/architecture/cloud-native/other-deployment-options.md [hunk 2]
```diff
@@ -13,15 +13,15 @@ To deploy to [Azure App Service for Containers](https://azure.microsoft.com/serv
...
 ![Azure Container Registry Run Instance](./media/acr-runinstance-contextmenu.png)
-**Figure 2-X**. Azure Container Registry Run Instance
+**Figure 2-17**. Azure Container Registry Run Instance
 
-Creating a container instance from the registry just requires specifying the usual Azure settings (name, subscription, resource group, and location) as well as how much memory to allocate to the container and which port it should listen on. Figure 2-X shows the Azure Portal form used to create the new instance.
+Creating a container instance from the registry just requires specifying the usual Azure settings (name, subscription, resource group, and location) as well as how much memory to allocate to the container and which port it should listen on. Figure 2-18 shows the Azure Portal form used to create the new instance.
 
 ![ACR Run Instance Create container instance](./media/acr-create-deeplink.png)
-**Figure 2-X**. ACR Run Instance Create container instance
+**Figure 2-18**. ACR Run Instance Create container instance
```
Comment (rpetrusha) [inline docs/architecture/cloud-native/other-deployment-options.md]:
```suggestion
**Figure 2-18**. ACR Run Instance Create container instance
```

File: docs/architecture/cloud-native/other-deployment-options.md [hunk 3]
```diff
@@ -13,15 +13,15 @@ To deploy to [Azure App Service for Containers](https://azure.microsoft.com/serv
...
 
 ![Azure Container Registry Run Instance](./media/acr-runinstance-contextmenu.png)
-**Figure 2-X**. Azure Container Registry Run Instance
+**Figure 2-17**. Azure Container Registry Run Instance
 
-Creating a container instance from the registry just requires specifying the usual Azure settings (name, subscription, resource group, and location) as well as how much memory to allocate to the container and which port it should listen on. Figure 2-X shows the Azure Portal form used to create the new instance.
+Creating a container instance from the registry just requires specifying the usual Azure settings (name, subscription, resource group, and location) as well as how much memory to allocate to the container and which port it should listen on. Figure 2-18 shows the Azure Portal form used to create the new instance.
 
 ![ACR Run Instance Create container instance](./media/acr-create-deeplink.png)
```
Comment (rpetrusha) [inline docs/architecture/cloud-native/other-deployment-options.md]:
```suggestion
![ACR Run Instance Create container instance](./media/acr-create-deeplink.png) \
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
There are two cases where the figure caption appears on the same line as the figure rather than below. My suggested changes should fix the issue, @ardalis.

**Comment by ardalis** [comment]
Hopefully good to merge now, thanks!

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
I fixed one more occurrence, @ardalis. I'll merge now.

**Pull request merged**